test/classes/delete-free/lifetimes/forall-of-new-borrowed-escapes.bad

```diff
@@ -0,0 +1,11 @@
+forall-of-new-borrowed-escapes.chpl:13: warning: result of new C is now managed by default
+forall-of-new-borrowed-escapes.chpl:13: note: 'new unmanaged C' gives old behavior
+forall-of-new-borrowed-escapes.chpl:13: note: 'new borrowed C' is the new default
+forall-of-new-borrowed-escapes.chpl:13: note: 'new owned C', 'new shared C' also available
+forall-of-new-borrowed-escapes.chpl:13: note: get more help with --warn-unstable
+forall-of-new-borrowed-escapes.chpl:13: In iterator 'chpl__loopexpr_iter2':
+forall-of-new-borrowed-escapes.chpl:13: warning: result of new C is now managed by default
+forall-of-new-borrowed-escapes.chpl:13: note: 'new unmanaged C' gives old behavior
+forall-of-new-borrowed-escapes.chpl:13: note: 'new borrowed C' is the new default
+forall-of-new-borrowed-escapes.chpl:13: note: 'new owned C', 'new shared C' also available
+forall-of-new-borrowed-escapes.chpl:13: note: get more help with --warn-unstable
```

Comment:
I'd probably write `new borrowed` to avoid these warnings. Some things like `chpl__loopexpr_iter2` might not be stable over time. OK this way though if you prefer, for some reason. I wouldn't add `--no-warnings` to these.

Reply:
Oh, I hadn't read closely enough to notice the mention of the internal name...  That's too bad.  I liked writing them without the `borrowed` best because it seems like a case where people would be most surprised by this ("I wasn't thinking about ownership, so didn't think about these being borrowed") where the warning helps such people somewhat but if it went away before these checks were improved, I could imagine it causing confusion (since it did even for me who was thinking of them as explicitly being borrowed).  I'm inclined to leave them alone for now and if it becomes a maintenance burden before things improve, I'll migrate them to something better.